### DISCUSSION TITLE
Why can't we bind "Ctrl PageDown" in zellij?

### DISCUSSION THREAD

**Discussion by AlgusDark** [zellij-org/zellij#2579] [Q&A] [answered]
I use Ctrl+PageDown and PageUp in browsers and on other applications.
I tried to do a bind in normal mode in zellij and there is an error.

Why can't we do that? Are there any plans for adding this feature?

**Comment by tlinford** [comment, marked as answer]
ctrl+pagedown doesn't even get sent by terminal emulators, so not much that zellij can do there.
i'm not sure why exactly, but let's just say that keyboard input in terminals is a complex problem with many pitfalls :smile:  (http://www.leonerd.org.uk/hacks/fixterms/).

EDIT: i was wrong, i originally tested this inside vscode terminal and it didn't register there!

it seems it's actually termwiz that doesn't parse ctrl+pagedown.

**Reply by AlgusDark** [reply]
I was assuming something like that would be the reason 🤔 I do wonder what kind of tricks WezTerm use for that. Well, time to read.

Thanks 🙏

**Comment by Lockszmith-GH** [comment]
What OS? What Terminal?

**Reply by AlgusDark** [reply]
MacOS and WezTerm

**Reply by Lockszmith-GH** [reply]
I have zero experience with MacOs, so don't know WezTerm. I did however search the website for PageUp and found this:
https://wezfurlong.org/wezterm/config/lua/keyassignment/CopyMode/PriorMatchPage.html?h=page+up

My guess, the reason the keys aren't getting to Zellij, is because WezTerm has something bound to it.

**Reply by AlgusDark** [reply]
Thanks for taking a look. After reading the zellij docs, I found out that zellij only accepts alphabet with Ctrl. A cool think I found out with WezTerm is that I can change any combination to anything else. Eight now I'm bounding Ctrl+pagedown to Unicode chars I don't use and is working super good.
